packages/alert/__tests__/Alert.test.tsx

```diff
@@ -1,16 +1,29 @@
 import colors from '@littlespoon/theme/src/colors'
-import { render, screen } from '@testing-library/react'
+import { act, fireEvent, render, screen } from '@testing-library/react'
 
 import type { AlertProps } from '../src/'
 import Alert from '../src/'
 
+const onCloseMock = jest.fn()
+
 describe('no props', () => {
   it('renders alert with className', async () => {
     render(<Alert className="alert"></Alert>)
     expect(document.getElementsByClassName('alert')).toHaveLength(1)
   })
 })
 
+describe('onClose called on close button click', () => {
+  it('renders alert with className', async () => {
+    render(<Alert onClose={onCloseMock}></Alert>)
+    const btnClose = screen.getByTestId('btn__close')
+    act(() => {
+      fireEvent.click(btnClose)
+    })
+    expect(onCloseMock).toHaveBeenCalledTimes(1)
+  })
+})
+
```

Comment:
Unrelated to your changes, but it seems odd that we can create an alert with no title or description. I think at least one of the two should be provided. Wdyt?

Reply:
Design shows both fields are optional.

Reply:
I don't think that is correct. Based on ZeroHeight we always have a description. https://zeroheight.com/3ddd0f892/p/76cbba-alert/b/8982a2

At most, one should be optional if the other is present, but it seems to me that `description` is required, which makes sense because why show a Toast with no message? Where did you see that?

Reply:
ah, right.
Sorry, yeah the description is always there.